drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/ScenarioSimulationXMLPersistence.java

```diff
@@ -28,17 +40,34 @@
 
     private ScenarioSimulationXMLPersistence() {
         xt = XStreamUtils.createTrustingXStream(new DomDriver());
+
+        xt.autodetectAnnotations(true);
+
+        xt.alias("ExpressionElement", ExpressionElement.class);
```

Comment:
Just idea, couldn't we use 
```java
xt.alias(ExpressionElement.class.getSimpleNmae(), ExpressionElement.class);
```

Reply:
In general I agree with you but in this specific use case I suggest to use the string to avoid that a refactor will break existing scenario loading: the IDE will update class name without any warn but then you will be not able to reload an existing file

Reply:
thanks